### ISSUE TITLE
Create a single re-attempt for the attachment of a 10-10CG submission

### ISSUE THREAD

**Issue by kevinmirc** [department-of-veterans-affairs/va.gov-team#15147]
Out of ~8000 10-10cg submissions, 3 submissions did not have an attachment.

Based on the logs, it looks like the API never attempted to send an attachment because the generation of the PDF failed (see https://github.com/department-of-veterans-affairs/va.gov-team/issues/14429).

Since the PDF generation failure seems to happen randomly, consider implementing a single re-attempt to generate and send again. This can happen within the same req/res cycle and only when an error occurs. 

Prerequisite: https://github.com/department-of-veterans-affairs/va.gov-team/issues/15146

**Comment by kevinmirc** [comment]
Closing in favor of: https://github.com/department-of-veterans-affairs/va.gov-team/issues/16952
